sr_hand_config/common/src/sr_hand_config/load_diagnostic_analyzer.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2021 Shadow Robot Company Ltd.
+# Copyright 2021-2022 Shadow Robot Company Ltd.
```

Comment:
I think this should be 2021, 2022?

Reply:
no as we are making active changes to this repo between 2021 and 2022